Leetcode/529-Minesweeper.py

```diff
@@ -0,0 +1,33 @@
+from collections import deque
+class Solution:
+    def updateBoard(self, board: List[List[str]], click: List[int]) -> List[List[str]]:
+        dirs = [(0, 1), (1, 0), (1, 1), (-1, -1), (0, -1), (-1, 0), (1, -1), (-1, 1)]
+        m, n = len(board), len(board[0])
+
+        queue = deque([click])
+        visited = set([click[0], click[1]])
```

Comment:
```Suggestion
visited = set([(click[0], click[1])])
```
This would seem the right implementation but i'm not sure what this line is supposed to do?